app-web/src/components/Search/index.js

```diff
@@ -49,9 +49,10 @@ export const Search = ({ onSearch, searchOnEnter, inputConfig, query, ...rest })
 
   const search = () => {
     onSearch(terms);
-    setTerms('');
   };
 
+  const handleFocus = event => event.target.select();
```

Comment:
what does this do?

Reply:
This will allow single click and select all text in input box

Reply:
hmm I'm not sure if that will be an effect we will want 100% of the time when searching. Just my opinion. @sheaphillips ?

I feel like if you have 'searched' than a single click highlight would be an awesome UX improvement. If it happens always, it prevents users from 'fixing typos'

Reply:
Also as a tip, i'd add a tiny comment above that event handler that explains exactly what the function does  :)